go.mod

```diff
@@ -7,10 +7,10 @@ require (
 	github.com/ahmetb/gen-crd-api-reference-docs v0.3.0
 	github.com/coreos/go-systemd/v22 v22.5.0
 	github.com/gardener/etcd-druid v0.22.0
-	github.com/gardener/gardener v1.92.3
+	github.com/gardener/gardener v1.94.1
```

Comment:
This should be the k8s-1.30-fix fork

Reply:
We need to release the regular version and then do a dedicated build for the 1.30 compatibility. As we did for the other releases, too.

Reply:
This is true !